### ISSUE TITLE
implement colorationSpan in MEI 4.0

### ISSUE THREAD

**Issue by craigsapp** [music-encoding/music-encoding#500]
It would be useful to add a `colorationSpan` element which would function similarly to the new `ligatureSpan` element:
     https://github.com/music-encoding/music-encoding/commit/3d000618562167b4a82d5956311d535896e7fab6

`ligatureSpan` elements are used to indicate where `ligature` was used in the non-CMN original notation.  Likewise `colorationSpan` would indicate where a `coloration` was used in the non-CMN original notation (or whatever you are calling `coloration` in the neumes/mensuration part of MEI).  For example if the name is `color`, then of course it would be called `colorSpan`.  If there is no name for it in MEI, then `colorationSpan` is probably better as otherwise people will be confused about the meaning of the word "color".

I put an example in either the music-encoding or verovio issues for coloration markers, but can't find it at the moment.  But here is a sample:

<img width="504" alt="screen shot 2018-02-16 at 8 20 29 pm" src="https://user-images.githubusercontent.com/3487289/36337912-9e3846f8-1356-11e8-8533-a521d7304068.png">

The solid square bracket above the notes is a `ligatureSpan` and the dashed square bracket above the second note is a proposed `colorationSpan`.  

Ligatures always have to have two or more notes in them.  Coloration can be applied to one or more notes.

**Comment by rettinghaus** [comment]
I've seen that coming … 
I still think `<bracketSpan func="coloration">` and `<bracketSpan func="ligature">` would have been a better approach. 
See #403

**Comment by craigsapp** [comment]
Either way will work for me.

**Comment by pe-ro** [comment]
`<bracketSpan>` added and `<ligatureSpan>` removed by 39d5e46

**Comment by craigsapp** [comment]
@axgeertinger wrote:
>  So, `<ligature>` is no longer supported in MEI 4.0.0; instead, `<bracketSpan func='ligature'>` is introduced. 

No, I don't think so, but @pe-ro can clarify:  `<ligature>` is for encoding music that contains a visual ligature (as in the originally notated music).  The `<bracketSpan func='ligature'>` is used to encompass a series of individual (modern) notes that were originally in a ligature, but now are notated as separate notes, and the original intent is indicated by a bracket in the notation.

In other words `<ligature>` is specific to the mensural module of MEI and would be used to encode notes that are visually stuck together (which in turn defines their individual durations), and `<bracketSpan func='ligature'>` would be used to encode the same music in the cmn module as modern separate notes.

**Comment by axgeertinger** [comment]
Ah, now I see it! Yes, indeed `<ligature>`is still available in the mensural module. My apologies for missing that.
